src/site/content/en/blog/script-component/index.md

```diff
@@ -0,0 +1,221 @@
+            <td rowspan="2" >Script component with strategy = afterInteractive for both scripts
+            </td>
+        </tr>
+        <tr>
+            <td>Twitter Follow button without async or defer
+            </td>
+        </tr>
+    </tbody>
+    <caption>Script and Script Component configuration for demo 1 with 2 scripts.</caption>
+</table>
+</div>
+
+
+
+The following comparison shows the visual progress for both versions of the [Next.js commerce starter-kit](https://demo.vercel.store/). As seen, LCP occurs nearly 1s earlier with the Script component enabled with the right loading strategy.
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/a3gpB5Ji7vcvQjBhviCP.png", alt="Film strip comparison showing improvment in LCP", width="800", height="372" %}
+
+### Third-party scripts in a Next.js blog
+
+Third-party scripts were added to the demo blog app as given below. 
+
+<div>
+<table>
+    <thead>
+        <tr>
+            <th>Before</th>
+            <th>After</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Google Tag Manager with async</td>
+            <td rowspan="2" >Script component with strategy = lazyonload for each of the four scripts
+            </td>
+        </tr>
+        <tr>
+            <td>Twitter Follow button with async
+            </td>
+        </tr>
+        <tr>
+            <td>YouTube Subscribe button without async or defer
+            </td>
+        </tr>
+        <tr>
+            <td>LinkedIn Follow button without async or defer
+            </td>
+        </tr>
+    </tbody>
+    <caption>Script and Script Component configuration for demo 2 with 4 scripts.</caption>
+</table>
+</div>
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/BAWQVlyBX1UUJOMyrBJm.gif", alt="Video showing loading progress with and without the Script component ", width="800", height="306" %}
+
+As seen in the video, FCP occurs at 0.4s on the page without the Script component and 0.9 sec with the Script component. 
+
+
+## What’s next for the Script component
+
+While the strategy options for `afterInteractive` and `lazyOnload` provide significant control over render-blocking scripts, we are also exploring other options that would increase the utility of the Script component. 
+
+
+### Using web workers
+
+[Web workers](https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers) can be used to run independent scripts on background threads which can free up the main thread to handle processing user interface tasks and improve performance. Web Workers are best suited for offloading JavaScript processing, rather than UI work, off the main thread. Scripts used for customer support or marketing, which typically do not interact with the UI, may be good candidates for execution on a background thread. A lightweight third-party library - [PartyTown](https://github.com/BuilderIO/partytown) may be used to isolate such scripts into a web worker.
```

Comment:
```suggestion
[Web workers](https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers) can be used to run independent scripts on background threads which can free up the main thread to handle processing user interface tasks and improve performance. Web Workers are best suited for offloading JavaScript processing, rather than UI work, off the main thread. Scripts used for customer support or marketing, which typically do not interact with the UI, may be good candidates for execution on a background thread. A lightweight third-party library&mdash;[PartyTown](https://github.com/BuilderIO/partytown)&mdash;may be used to isolate such scripts into a web worker.
```